internal/web/routes/routes.go

```diff
@@ -0,0 +1,18 @@
+package routes
+
+import (
+	"context"
+
```

Comment:
Check for nil pointers on githubClient or openAIClient before using them in GithubController instantiation. If either client is nil, it may cause a panic when the methods are called.